app/controllers/api/verify/complete_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::Verify::CompleteController < Api::BaseController
```

Comment:
with a few exceptions* we mostly use the nested namespace way to define classes:
```suggestion
module Api
  module Verify
    class CompleteController < Api::BaseController
```

One reason to prefer the indented style is that it means the files can be loaded in any order. If this file was loaded before the `Api::Verify` module was defined, it could cause an error, but the nested style reopens namespaces each time so it's more flexible.

---

*exceptions:
```
> git grep -E "class.*::.* <" -- '*.rb'  | wc -l
       7
```